python/travis/run_python_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#
+#  Copyright 2015-2017 TWO SIGMA OPEN SOURCE, LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+set -ev
+
+source activate flint
+
+PYSPARK_PYTHON=python3 spark/bin/spark-submit --master=local --jars ../target/scala-2.11/flint-assembly-0.2.0-SNAPSHOT.jar --py-files ../target/scala-2.11/flint-assembly-0.2.0-SNAPSHOT.jar test/test_dataframe.py
```

Comment:
@mforsyth , thx for the heads up.

There are a few requirements I want to clarify:
1. Have a sparkContext, pyspark provider interface. Internally we have sth like:

```
def pyspark(launcher):
    return launcher.import_pyspark()

def sc(launcher, pyspark):
    with test_support.suppress_instrumentation:
        with launcher.create_spark_context() as sc:
            yield sc
    from pyspark import SparkContext
    SparkContext._gateway.shutdown()
    SparkContext._gateway = None
    reset_env()
```
I'd imagine in the open source land we probably have another provider that knows how to import pyspark and create context.